src/App.js

```diff
@@ -1,12 +1,69 @@
-import React from 'react';
+import React, {useEffect} from "react";
 import './App.css';
 
+import Pokecard from "./Components/Pokecard";
+import axios from "axios";
+
+//previous = result.data.null
+//next = result.data.next
+
+
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokemon, setPokemon] = React.useState('');
```

Comment:
Ik zou de default waarde aanpassen naar `undefined`. Wanneer de waarde geset is met een object dan veranderd het type van string naar object, wat ik niet zo netjes vind. Wanneer het undefined is, is het ook duidelijker wat het zou kunnen betekenen: namelijk dat het nog niet is in geladen. Je check onderin kan dan ook iets simpeler